server/integration/infinispan/src/main/java/org/infinispan/server/infinispan/task/InfinispanIntegrationMetadataFileFinder.java

```diff
@@ -0,0 +1,19 @@
+package org.infinispan.server.infinispan.task;
+
+import org.infinispan.factories.components.ModuleMetadataFileFinder;
+import org.kohsuke.MetaInfServices;
+
+
+/**
+ * Author: Michal Szynkiewicz, michal.l.szynkiewicz@gmail.com
+ * Date: 1/22/16
+ * Time: 2:21 PM
+ */
+@MetaInfServices
+public class InfinispanIntegrationMetadataFileFinder implements ModuleMetadataFileFinder {
+
+   @Override
+   public String getMetadataFilename() {
+      return "infinispan-server-infinispan-component-metadata.dat";
```

Comment:
this is ${basedir}/src/main/resources/${project.artifactId}-component-metadata.dat.

The artifactId for server/integration/infinispan is: infinispan-server-infinispan